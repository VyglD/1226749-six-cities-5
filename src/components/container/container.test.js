import React from "react";
import renderer from "react-test-renderer";
import Container from "./container";

it(`Render correctly component Container`, () => {
  const component = renderer.create(
      <Container
        id={`id`}
        style={{}}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
