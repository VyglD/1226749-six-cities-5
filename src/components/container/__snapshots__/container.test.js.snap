// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly component Container 1`] = `
<div
  id="id"
  style={Object {}}
/>
`;
